packages/lib/src/utils/detectInIframeInAccessibleDomain.ts

```diff
@@ -0,0 +1,7 @@
+// Returns true if the page is being run in an iframe (in a domain we have access to)
+export default () => {
+    if (window.parent.location instanceof Location) {
```

Comment:
Does this check mean the adyen-web is in an iframe which is a different domain than the application?